Tasks/google-play-release/GooglePlay.ts

```diff
@@ -977,11 +954,12 @@ function getAllApkPaths(mainApkFile: string): string[] {
         tl.debug(`Additional APK pattern: ${additionalApk}`);
         let apkPaths: string[] = resolveGlobPaths(additionalApk);
 
-        apkPaths.forEach((apkPath) => {
+        apkPaths.forEach(async (apkPath) => {
```

Comment:
This doesn't wait for the async methods to complete. I think apkFileList will be complete, but it would probably be less surprising if you used `await Promise.all(apkPaths.map(async (apkPath) => {` here.

Reply:
Good catch!

Reply:
Or as we discussed, make this code simple and synchronous by switching over to for loops.

Reply:
Per our off-line discussion I'm making that change.  If nothing else, to make sure that the messages don't get interleaved.